src/transaction/log_manager.c

```diff
@@ -10889,9 +10889,13 @@ log_clock_daemon_init ()
 void
 log_check_ha_delay_info_daemon_init ()
 {
-  bool do_supplemental_log = prm_get_integer_value (PRM_ID_SUPPLEMENTAL_LOG) > 0 ? true : false;
+  const bool do_supplemental_log = prm_get_integer_value (PRM_ID_SUPPLEMENTAL_LOG) > 0 ? true : false;
+  const bool do_calc_replication_delay = prm_get_bool_value (PRM_ID_ER_LOG_CALC_REPL_DELAY);
+  const bool is_transaction_server = (get_server_type () == SERVER_TYPE_TRANSACTION);
```

Comment:
```suggestion
  const bool is_transaction_server = is_transaction_server();
```

Reply:
or `is_active_transaction_server()` though it is only called when it's not PTS?

Reply:
It will be better to use `is_active_transaction_server()` for more readability. 
I used `(get_server_type () == SERVER_TYPE_TRANSACTION)` just because caller checks whether it is passive transaction server or not.